### DISCUSSION TITLE
youtube embed

### DISCUSSION THREAD

**Discussion by jformela** [directus/directus#11471] [Archive]
Hi,
any chance to have API connection to my YouTube channel so I can pick the video by selecting its thumbnail or title and store YT_video_id in my directus collection data?

There is a mention about configuring YT API key in v8 documentation, however no info regarding this one in current version.
https://v8.docs.directus.io/guides/files.html#whitelisting-thumbnails

**Comment by rijkvanzanten** [comment]
There's no native youtube video picker field or anything like that. You might be able to rely on the API interface though, which does a very similar thing to any arbitrary API 👍🏻
